deployment/Dockerfile

```diff
@@ -20,15 +29,19 @@ RUN cargo chef prepare --recipe-path recipe.json
 FROM chef as builder
 ENV CARGO_NET_GIT_FETCH_WITH_CLI=true
 COPY --from=planner /build/recipe.json recipe.json
+
 # Build our project dependecies, not our application!
-RUN cargo chef cook --release --no-default-features --features "production" -p fuel-core-bin --recipe-path recipe.json
+RUN xx-cargo chef cook --release --no-default-features --features "production" -p fuel-core-bin --recipe-path recipe.json
 # Up to this point, if our dependency tree stays the same,
 # all layers should be cached.
 COPY . .
-RUN cargo build --release --no-default-features --features "production" -p fuel-core-bin
+RUN xx-cargo build --release --no-default-features --features "production" -p fuel-core-bin \
+    && xx-verify ./target/$(xx-cargo --print-target-triple)/release/fuel-core \
+    && mv ./target/$(xx-cargo --print-target-triple)/release/fuel-core ./target/release/fuel-core \
+    && mv ./target/$(xx-cargo --print-target-triple)/release/fuel-core.d ./target/release/fuel-core.d
```

Comment:
I don't think so, since the final docker image only copies the resultant binary over and leaves the target folder behind from the build stage image. It may even degrade the caching performance during the build stage if we remove it, but I haven't tested that.

Reply:
goooood